core/src/main/scala/cats/Foldable.scala

```diff
@@ -582,6 +582,32 @@ import Foldable.sentinel
 
   override def unorderedFoldMap[A, B: CommutativeMonoid](fa: F[A])(f: (A) => B): B =
     foldMap(fa)(f)
+
+  /**
+   * Partitions elements in fixed size.
+   * Returns empty `F` if `size <= 0`.
+   *
+   * Example:
+   * {{{
+   * scala> import cats.implicits._
+   * scala> val x: List[Int] = List(1, 2, 3, 4, 5)
+   * scala> Foldable[List].grouped(x)(2)
+   * res0: List[List[Int]] = List(List(1, 2), List(3, 4), List(5))
+   * }}}
+   */
+  def grouped[A](fa: F[A])(size: Int)(implicit F: Alternative[F]): F[F[A]] =
+    // Need to check on empty as otherwise the final combine would return F[Empty] rather than just Empty
+    if (size <= 0 || isEmpty(fa)) {
+      F.empty
+    } else {
```

Comment:
Potentially we could also check for `size = 1` and just map using `pure`, though not sure why someone would do that :smile: